components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -3383,6 +3410,10 @@ private class ConfigElements {
         private static final String DEVICE_CODE_EXPIRY_TIME = "ExpiryTime";
         private static final String DEVICE_CODE_POLLING_INTERVAL = "PollingInterval";
         private static final String DEVICE_CODE_KEY_SET = "KeySet";
+
+        // Allow Cross Tenant Introspection Config
+        public static final String ALLOW_CROSS_TENANT_TOKEN_INTROSPECTION = "AllowCrossTenantTokenIntrospection";
```

Comment:
Should be a private constant since this config name won't be used by external parties

Reply:
Resolved by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1846/commits/21769f73d296f354b16b51e619989d93b4d75804